packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -162,13 +163,15 @@ class UnreferencedStateTracker {
         currentTimestampMs: number,
         deleteTimeoutMs: number,
         inactiveNodeId: string,
+        packagePath?: readonly string[],
     ) {
         if (this.inactive && !this.inactiveEventsLogged.has(eventName)) {
             logger.sendErrorEvent({
                 eventName,
                 age: currentTimestampMs - this.unreferencedTimestampMs,
                 timeout: deleteTimeoutMs,
                 id: inactiveNodeId,
+                pkg: { value: packagePath ? `/${packagePath.join("/")}`: undefined, tag: TelemetryDataTag.PackageData },
```

Comment:
if id here means "internal" id (i.e. only guids, after we switch to aliasing as a way to create roots), then it's fine.
Otherwise, same TelemetryDataTag.PackageData applies to ID
Can you please check that we do not drop such properties today (with such tag)?

Reply:
Yep, id should always be internal. And yes, we do not drop PackageData tagged properties.

Reply:
@markfields FYI